hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -2087,6 +2087,20 @@
     <name>yarn.nodemanager.linux-container-executor.cgroups.mount-path</name>
   </property>
 
+  <property>
+    <description>This property sets the mount path for CGroups v2.
+      This parameter is optional, and needed to be set only in mixed mode,
+      when CGroups v2 is mounted in a different path than Cgroups v1.
+      For example, in hybrid mode, CGroups v1 controllers can be mounted in
+      /sys/fs/cgroup, while v2 can be mounted in /sys/fs/cgroup/unified folder.
```

Comment:
> v1 controllers can be mounted in /sys/fs/cgroup

Might be a bit misleading in this format, maybe say something like this:
"v1 controllers can be mounted under /sys/fs/cgroup/ (for example /sys/fs/cgroup/cpu,cpuacct)". Otherwise it can seem like the cgroup v2 FS is mounted under a cgroup v1 FS.

Reply:
It happened to be a bit misleading indeed, I fixed the phrasing. Thanks for the suggestion!